src/Parser/Shaman/Elemental/Modules/Talents/ExposedElements.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import enemy from 'Parser/Core/Entity';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatNumber, formatPercentage } from 'common/format';
+import SpellIcon from 'common/SpellIcon';
+
+import SPELLS from 'common/SPELLS/SHAMAN';
+
+import { EXPOSED_ELEMENTS } from 'Parser/Shaman/Elemental/Constants';
+
+class ExposedElements extends Analyzer {
+  removeDebuffTimestamp = null;
+  damageGained=0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.EXPOSED_ELEMENTS_TALENT.id);
+  }
+
+  on_byPlayer_cast(event){
+    if(this.removeDebuffTimestamp === null)
```

Comment:
please always use curly brackets to enclose statements, even if it is one liners